auditbeat/auditbeat.reference.yml

```diff
@@ -250,6 +250,14 @@ auditbeat.modules:
 #       - from: "a"
 #         to: "b"
 #
+# The following example use dissect syntax to tokenize the string into fields:
```

Comment:
Can you just say, "The following example tokenizes the string into fields:"  That way, if you change the name of the processor someday, you won't have to update the description. :-)

If you feel you _need_ to mention dissect syntax, then say, "The following example uses..." (plural).

Reply:
:D